xdebug_com.c

```diff
@@ -26,6 +26,7 @@
 # include <sys/poll.h>
 # include <unistd.h>
 # include <sys/socket.h>
+# include <sys/un.h>
```

Comment:
Is this a header that is available on Windows too? As it's not being used there, it probably should go in an ``#ifdef`` as well.